bittensor/dendrite.py

```diff
@@ -705,7 +706,7 @@ def process_server_response(
                     # Set the attribute in the local synapse from the corresponding
                     # attribute in the server synapse
                     setattr(local_synapse, key, getattr(server_synapse, key))
-                except:
+                except Exception:
```

Comment:
What are your thoughts on making a custom exception get raised here 

```suggestion
                except ServerResponseError as e:
                # logging
                raise
```